pre-commit/BUILD.bazel

```
@@ -216,3 +216,31 @@ sh_binary(
     ],
     visibility = ["//visibility:public"],
 )
+
+sh_test(
+    name = "buf-breaking",
+    size = "small",
+    srcs = ["check-incompatibilities.sh"],
+    data = [
+        "//:WORKSPACE.bazel",
+        "//:buf.yaml",
+        "@buf",
+    ],
+    env = {
+        # Hack to escape the sandbox and grep on the actual repository
+        "WORKSPACE": "$(rootpath //:WORKSPACE.bazel)",
+        "buf_config": "$(rootpath //:buf.yaml)",
+        "buf_path": "$(rootpath @buf//:buf)",
```

Why not:
```suggestion
        "BUF": "$(rootpath @buf//:buf)",
```
and skip the `BUF="$(readlink "$buf_path")"` in `pre-commit/check-incompatibilities.sh` ?

Same question for `buf_config`.

---

Ah fair point. I copied existing code, not sure if it was written that way to work with pre-commit but I'll try it.

---

Hmm... I think the `readlink` might still be required, as I get this error now: https://dash.zh1-idx1.dfinity.network/invocation/12a095a0-f175-4cd4-a774-75474eab2ebc?target=%2F%2Fpre-commit%3Abuf-breaking&targetStatus=6